core/arch/arm/plat-hikey/platform_flags.mk

```diff
@@ -29,3 +29,5 @@ user_ta-platform-cflags += $(arm32-platform-cflags)
 user_ta-platform-cflags += -fpie
 user_ta-platform-cppflags += $(arm32-platform-cppflags)
 user_ta-platform-aflags += $(arm32-platform-aflags)
+
+CFG_HIKEY_UART3 ?= n
```

Comment:
Please remove this. This definition is not needed (since 'disabled' is the default for any `CFG_` variable), but it is also not good to have ir here because this file is exported in the "TA dev kit" (where it makes no sense).